scripts/taginfo.js

```diff
@@ -2,14 +2,47 @@
 
 import * as fs from "fs";
 import * as ShieldDef from "../src/js/shield_defs.js";
+import * as Shields from "../src/js/shield.js";
+import * as Gfx from "../src/js/screen_gfx.js";
+import * as CustomShields from "../src/js/custom_shields.js";
+import * as skia from "skia-canvas";
+import namer from "color-namer";
+import { mkdir } from "node:fs/promises";
+
+await mkdir("dist/shield-sample", { recursive: true });
+
+//Headless graphics context
+Gfx.setGfxFactory((bounds) => {
+  let canvas = new skia.Canvas(bounds.width, bounds.height);
```

Comment:
Wow, I was griping about potential image quality issues on 2× displays when downscaling PNGs, but I didn’t imagine that you were going to pull in Skia for a very correct fix. Thank you! If GL JS ever supports native vector graphics as sprites, we’ll be in a better position to set them up at build time.